### ISSUE TITLE
License updated to Gnu Lesser General Public License, version 3

### ISSUE THREAD

**Issue by adcroft** [NOAA-GFDL/MOM6#580]
As of August 24, 2017, commit 52b94dd5f274621b31cfe0b08e89dc73439a9667, the license shipped with MOM6 has been updated to version 3 of the Gnu Lesser General Public License. Permission to change the license was obtained from all contributors that had contributed under the previous license (as recorded by git and on GitHub) .

If you have a fork of MOM6 predating the change in license, you are not obliged to change the license. The new license only applies to versions of MOM6 newer than August 24, 2017 which have the new license file. You can keep the old license by not updating to our latest code. However, we ask that you accept the new license by updating to the latest code on the dev/master branch. We will not accept pull requests for contributions unless they are made under the new license.

In a nutshell: we replaced the GPLv3 license with the LGPLv3 license that allows MOM6 to be used within a coupled model that might not be using a compatible license. If you have questions, please reach out to us on this thread or by email.

**Comment by marshallward** [comment]
Ok to close this?
